### PR TITLE
add LRC Local Rademachar Complexity Regularization

### PR DESCRIPTION
**Pull request by jerrywgz**

### CODE REVIEW

File: fluid/AutoDL/LRC/genotypes.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            ('skip_connect', 0), ('sep_conv_3x3', 1)],
+    normal_concat=range(2, 6),
+    reduce=[('max_pool_3x3', 0), ('max_pool_3x3', 1), ('max_pool_3x3', 0),
+            ('skip_connect', 2), ('max_pool_3x3', 0), ('skip_connect', 2),
+            ('skip_connect', 2), ('skip_connect', 3)],
+    reduce_concat=range(2, 6))
+# MY_DARTS = Genotype(normal=[('dil_conv_3x3', 0), ('sep_conv_5x5', 1), ('skip_connect', 0), ('sep_conv_3x3', 2), ('skip_connect', 0), ('sep_conv_3x3', 1), ('skip_connect', 0), ('skip_connect', 1)], normal_concat=range(2,6), reduce=[('max_pool_3x3', 0), ('skip_connect', 1), ('max_pool_3x3', 0), ('dil_conv_5x5', 2), ('max_pool_3x3', 0), ('skip_connect', 3), ('skip_connect', 3), ('max_pool_3x3', 0)], reduce_concat=range(2, 6))
+# MY_DARTS = Genotype(normal=[('sep_conv_3x3', 0), ('sep_conv_3x3', 1), ('skip_connect', 0), ('sep_conv_3x3', 2), ('skip_connect', 0), ('sep_conv_3x3', 1), ('skip_connect', 0), ('skip_connect', 1)], normal_concat=range(2, 6), reduce=[('max_pool_3x3', 0), ('max_pool_3x3', 1), ('skip_connect', 2), ('sep_conv_3x3', 0), ('skip_connect', 3), ('skip_connect', 2), ('skip_connect', 3), ('skip_connect', 2)], reduce_concat=range(2, 6))
+
```
Comment (qingqing01) [inline fluid/AutoDL/LRC/genotypes.py:115]: Remove the unsed code

File: fluid/AutoDL/LRC/genotypes.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
+# Copyright (c) 2018 PaddlePaddle Authors. All Rights Reserved
```
Comment (qingqing01) [inline fluid/AutoDL/LRC/genotypes.py]: 2018 -> 2019

File: fluid/AutoDL/LRC/operations.py [hunk 3]
```diff
@@ -0,0 +1,346 @@
...
+        C_in,
+        kernel_size,
+        stride,
+        padding,
+        groups=C_in,
+        param_attr=ParamAttr(
+            initializer=Xavier(
+                uniform=False, fan_in=0),
+            name=name + 'op.1.weight'),
+        bias_attr=False)
```
Comment (qingqing01) [inline fluid/AutoDL/LRC/operations.py]: For the depthwise conv， set `use_cudnn=False` to speed

File: fluid/AutoDL/LRC/operations.py [hunk 4]
```diff
@@ -0,0 +1,346 @@
...
+        C_in,
+        kernel_size,
+        1,
+        padding,
+        groups=C_in,
+        param_attr=ParamAttr(
+            initializer=Xavier(
+                uniform=False, fan_in=0),
+            name=name + 'op.5.weight'),
+        bias_attr=False)
```
Comment (qingqing01) [inline fluid/AutoDL/LRC/operations.py]: same as above

File: fluid/AutoDL/LRC/run.sh [hunk 5]
```diff
@@ -0,0 +1,10 @@
+export LD_LIBRARY_PATH=/usr/lib64:/usr/local/lib:$LD_LIBRARY_PATH
```
Comment (qingqing01) [inline fluid/AutoDL/LRC/run.sh]: remove this line

**Review by qingqing01 — COMMENTED** [review]
In addition, need a README.md

File: fluid/AutoDL/LRC/genotypes.py (hunk 2 above)
Comment (jerrywgz) [reply fluid/AutoDL/LRC/genotypes.py]: Done

File: fluid/AutoDL/LRC/genotypes.py (hunk 1 above)
Comment (jerrywgz) [reply fluid/AutoDL/LRC/genotypes.py:115]: Done

File: fluid/AutoDL/LRC/operations.py (hunk 3 above)
Comment (jerrywgz) [reply fluid/AutoDL/LRC/operations.py]: Done, 430s/epoch -> 120s/epoch in V100

File: fluid/AutoDL/LRC/operations.py (hunk 4 above)
Comment (jerrywgz) [reply fluid/AutoDL/LRC/operations.py]: Done

File: fluid/AutoDL/LRC/run.sh (hunk 5 above)
Comment (jerrywgz) [reply fluid/AutoDL/LRC/run.sh]: Done

**Comment by jerrywgz** [comment]
> In addition, need a README.md

Done

**Review by qingqing01 — APPROVED** [review]

**Pull request merged**